helm-charts/konk-operator/dashboards/konk-operator.json

```diff
@@ -251,12 +265,57 @@
       "type": "row"
     },
     {
+      "alert": {
+        "alertRuleTags": {},
+        "conditions": [
+          {
+            "evaluator": {
+              "params": [
+                0
+              ],
+              "type": "lt"
```

Comment:
Is this checking for < 0 or <=0? Or is that not how the rule is read?

Reply:
`lt` means less than 0. The query returns -1 for not-ready pods, which will trigger this alert.